reconcile/utils/terrascript_aws_client.py

```diff
@@ -3490,11 +3490,22 @@ def _get_tf_resource_elasticsearch_log_groups(
 
         # res.get('', []) won't work, as publish_log_types is
         # explicitly set to None if not set
-        log_types = resource["publish_log_types"] or []
-        for log_type in log_types:
+        publish_log_types = resource["publish_log_types"] or []
+        for t in ElasticSearchLogGroupType:
+            log_type = t.value
+            if log_type not in publish_log_types:
+                publishing_options.append(
+                    {
+                        "log_type": log_type,
+                        "enabled": False,
```

Comment:
Would it make sense to have an explicit `"enabled": True` in corresponding line 3537?
I.e., if we activate a `log_type`, will TF automatically set it to `enabled` or will it keep the latest setting?

Reply:
TF will automatically set it to enable if cloudwatch_log_group_arn is not empty. But I agree to add it. This will make the code more readable.